pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: _discoveryapis_commons
-version: 0.1.8+2
+version: 0.1.8+3
```

Comment:
One could argue this is a new feature, not a bug-fix

Reply:
actually, that's a good point :)